Sources/Defaults/DefaultsBridge.swift

```diff
@@ -0,0 +1,185 @@
+import Foundation
+
+public protocol DefaultsBridge {
+	// The type of Value of Key<Value>
+	associatedtype Value
+
+	// This type should be one of the NativelySupportedType
+	associatedtype Serializable
+
+	// Serialize Value to Serializable before we store it in UserDefaults
+	func serialize(_ value: Value?) -> Serializable?
+
+	// Deserialize Serializable to Value
+	func deserialize(_ object: Serializable?) -> Value?
+}
+
+public struct DefaultsURLBridge: DefaultsBridge {
```

Comment:
Can you nest these inside `extension Defaults {}`? So they're "kinda" namespaced.